src/main/java/io/vlingo/actors/Grid.java

```diff
@@ -79,6 +90,16 @@ public void terminate() {
     world().terminate();
   }
 
+  @Override
+  public void quorumAchieved() {
```

Comment:
@buritos If these values are being set by a thread other the the reader of the same values, there's a strong chance that the changes will not be seen since the variables are not volatile/atomic.

Reply:
`hasQuorum` is volatile.

Reply:
@buritos Great, thanks. Now I see it. That's what happens by reviewing on my phone.